src/main/webapp/Portal.css

```diff
@@ -847,6 +847,10 @@ a.shortBtn {
 	margin-bottom: 0px !important;
 }
 
+.margin-bottom-5-imp {
```

Comment:
don't like to see new !important styles.  if the margin-bottom-5 is not working, then let's define a more specific selector.